.changeset/gentle-beds-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/theme-language-server-common': minor
```

Comment:
We could add theme-check-vscode: minor here so that this changeset appears in the VS Code extension marketplace changelog